File: Server/Domain/Message.cs
using System.Text.Json.Serialization;

namespace Domain;

public class Message
{
    public int Id { get; set; }
    public string Content { get; set; }

    public string Created { get; set; }
    
    public bool Sent { get; set; }

    public Message(int id, string content, string created, bool sent)
    {
        Id = id;
        Content = content;
        Created = created;
        Sent = sent;
    }
}

File: Android/app/src/main/java/com/example/android/entities/Message.java
package com.example.android.entities;

public class Message
{
    public int id;
    public String content;
    public String created;
    public Boolean sent;

    public Message(int id, String content, String created, Boolean sent)
    {
        this.id = id;
        this.content = content;
        this.created = created;
        this.sent = sent;
    }
}
